CMakeLists.txt

```diff
@@ -78,6 +85,7 @@ endif()
 # Optimizations
 #
 set(OPTIMIZE "portable" CACHE STRING "Optimization and Tuning (set to off, portable, native, legacy, fastbuild)")
+set(FAST_MATH true CACHE BOOL "Use -fast-math optimization")
```

Comment:
Why not use `option`? Also, can you add a comment why this can be disabled? And please add that it's gcc/clang only to the help string.